src/main/java/org/alfresco/repo/policy/BehaviourFilter.java

```diff
@@ -67,23 +68,27 @@
     /**
      * @deprecated Since 4.0 use {@link #enableBehaviour(NodeRef)}
      */
+    @Auditable(parameters = { "nodeRef" })

```

Comment:
Are you sure you need to @Auditable? I would not expect that to be the case. Can you try removing this and running the tests again.

Reply:
Indeed @Auditable is not required, has been removed in [commit](https://github.com/Alfresco/alfresco-repository/pull/791/commits/d798ce02ae0e7c0ceb2a8eb6bf207bfbf8c72093).